### PR TITLE
Initial Pseudocode for RX Path

### PR DESCRIPTION
**Pull request by nibanks**

### CODE REVIEW

**Comment by maolson-msft** [comment]
I think it makes more sense to put this in the Receiving Packets section than in the Appendix, don't you?

File: quic-encryption-offload.md [hunk 1]
```diff
@@ -419,6 +419,49 @@ The steps are detailed [here](https://www.rfc-editor.org/rfc/rfc9001#name-header
...
+## Psuedocode for Receive Path
+
+```c++
+enum Action {
+    Continue, // Pass packet up
+    Drop
+};
+
+Action ProcessUdpPacket(_Inout_ Packet* packet) {
+  if (!packet->IsQuicShortHeader()) return Continue; // Not QUIC short header packet
```
Comment (maolson-msft) [inline quic-encryption-offload.md:431]: Two-space indents? This is heresy.

File: quic-encryption-offload.md [hunk 2]
```diff
@@ -419,6 +419,49 @@ The steps are detailed [here](https://www.rfc-editor.org/rfc/rfc9001#name-header
...
+  
+  uint8_t CidLength = ConnectionIdLengthTable.Lookup(packet->DestinationIpAddress(), packet->DestinationUdpPort());
```
Comment (maolson-msft) [inline quic-encryption-offload.md:433]: Woah, so we need two LUTs?!

Comment (nibanks) [reply quic-encryption-offload.md]: Yeah, as far as I can tell.

**Pull request merged**